js/__internal/grids/grid_core/filter_panel/module.ts

```diff
@@ -28,6 +28,8 @@ const FilterPanelView = modules.View.inherit({
 
   init() {
     this.getController('data').dataSourceChanged.add(() => this.render());
+
+    this._columnsController = this.component.getController('columns');
```

Comment:
Let's use ```this.getController``` instead of the ```this.component.getController```:

```suggestion
    this._columnsController = this.getController('columns');
```